app/assets/stylesheets/config/_bootstrap_variables.scss

```diff
@@ -26,7 +26,11 @@ $border-radius-sm: 2px;
 $body-bg: $bg;
 $body-color: white;
 
-#game-index, #card{
+#game-index {
+    background-color: $mizu;
+}
+
+#inputs .card{
```

Comment:
Had to change the id #card to something more explicit for channel broadcasting purposes (it's now #input-@input.id) So I corrected the css here @hiroya210 so that the cards remain blue!